bots/jabber_mirror_backend.py

```diff
@@ -298,7 +298,7 @@ def config_error(msg):
 
 if __name__ == '__main__':
     parser = optparse.OptionParser(epilog=
-'''Most general and Jabber configuration options may also be specified in the
+                                   '''Most general and Jabber configuration options may also be specified in the
 zulip configuration file under the jabber_mirror section (exceptions are noted
```

Comment:
The following lines of text could also be indented, even though the E128 rule doesn't require it - at the moment it's a bit hard to read those.